be1-go/.gitignore

```
@@ -6,4 +6,25 @@ coverage.html
 configServer1.json
 configServer2.json
 configServer3.json
-configServer4.json
\ No newline at end of file
+configServer4.json
+/internal/handler/answer/manswer/testdata/
+/internal/handler/method/broadcast/mbroadcast/testdata/
+/internal/handler/method/catchup/mcatchup/testdata/
+/internal/handler/method/getmessagesbyid/mgetmessagesbyid/testdata/
+/internal/handler/method/greetserver/mgreetserver/testdata/
+/internal/handler/method/heartbeat/mheartbeat/testdata/
+/internal/handler/method/publish/mpublish/testdata/
+/internal/handler/method/subscribe/msubscribe/testdata/
+/internal/handler/method/unsubscribe/munsubscribe/testdata/
+/internal/handler/message/mmessage/testdata/
+/internal/handler/channel/authentication/mauthentication/testdata/
+/internal/handler/channel/chirp/mchirp/testdata/
+/internal/handler/channel/coin/mcoin/testdata/
+/internal/handler/channel/consensus/mconsensus/testdata/
+/internal/handler/channel/election/melection/testdata/
+/internal/handler/channel/lao/mlao/testdata/
+/internal/handler/channel/reaction/mreaction/testdata/
+/internal/handler/channel/testdata/
+/internal/handler/channel/coin/hcoin/testdata/
+/internal/validation/protocol/
+/internal/handler/channel/root/mroot/testdata/
```

Using `**` pattern, it could be easier to maintain the .gitignore.
Unless there is a testdata/ directory that shouldn't be ignored ?
```suggestion
/internal/handler/**/testdata/
/internal/validation/protocol/
```

---

thx for the suggestion

---

And I also fixed the import name in all the project